tensorflow/contrib/distributions/python/kernel_tests/bijectors/softplus_test.py

```diff
@@ -43,9 +44,10 @@ def _softplus_ildj_before_reduction(self, y):
 
   def testHingeSoftnessZeroRaises(self):
     with self.cached_session():
-      bijector = Softplus(hinge_softness=0., validate_args=True)
-      with self.assertRaisesOpError("must be non-zero"):
-        bijector.forward([1., 1.]).eval()
+      with self.assertRaisesWithPredicateMatch(errors.InvalidArgumentError,
+                                               "must be non-zero"):
+        bijector = Softplus(hinge_softness=0., validate_args=True)
```

Comment:
Unused variable 'bijector' [unused-variable]

Reply:
Changed `bijector` to `_`.